### DISCUSSION TITLE
NuxtLink does not redirect when clicked

### DISCUSSION THREAD

**Discussion by niklasfjeldberg** [nuxt/nuxt#19776] [Questions]
I am unsure how to tackle this issue. When I add a NuxtLink component, it does not work when clicked. The HTML looks like it is correctly generated, but it does nothing when clicked. A normal <a> will work. Does anyone have any ideas on how I should tackle this problem?

I am using the latest Nuxt version, `3.3.1` with the following packages.

```
"devDependencies": {
  "@nuxtjs/eslint-config-typescript": "^12.0.0",
  "@nuxtjs/eslint-module": "^4.0.2",
  "@nuxtjs/google-fonts": "^3.0.0-1",
  "@nuxtjs/strapi": "^1.8.1",
  "@nuxtjs/stylelint-module": "^5.0.1",
  "@vueuse/nuxt": "^9.9.0",
  "eslint": "^8.36.0",
  "nuxt": "^3.3.1",
  "nuxt-icon": "^0.3.2",
  "nuxt-meilisearch": "^0.1.7",
  "sass": "^1.58.2",
  "sass-loader": "^13.2.0",
  "stylelint": "^15.3.0",
  "stylelint-config-standard-scss": "^7.0.1"
},
  "dependencies": {
  "@nuxtjs/i18n": "^8.0.0-beta.10",
  "@pinia/nuxt": "^0.4.7",
  "markdown-it": "^13.0.1",
  "normalize.css": "^8.0.1",
  "pinia": "^2.0.33",
  "qs": "^6.11.0",
  "vue-multiselect": "^3.0.0-beta.1"
}
```

I am unable to create a minimal working example.

**Comment by Kasopej** [comment]
I have these problems too sometimes. I suspect it is hydration breaking the reactive dom structure

https://www.lichter.io/articles/vue-hydration-error/
